pom.xml

```diff
@@ -7,7 +7,7 @@
 
     <groupId>org.testcontainers</groupId>
     <artifactId>testcontainers-jooq-codegen-maven-plugin</artifactId>
-    <version>0.0.3</version>
+    <version>0.0.4</version>
```

Comment:
Let's use `0.0.4-SNAPSHOT` for the beginning

Reply:
Done